core/src/test/scala/org/apache/spark/SparkFunSuite.scala

```diff
@@ -107,6 +108,15 @@ abstract class SparkFunSuite
     getTestResourceFile(file).getCanonicalPath
   }
 
+  protected final def copyAndGetResourceFile(fileName: String, suffix: String): File = {
+    val url = Thread.currentThread().getContextClassLoader.getResource(fileName)
+    // Copy to avoid URISyntaxException during accessing the resources from the dependent module.
```

Comment:
nit: "To avoid illegal accesses to a resource file inside jar (`URISyntaxException` might be thrown when accessing it), copy it into a temporary one for accessing it from the dependent module."

Reply:
> nit: How about rephrasing it like "To avoid illegal accesses to a resource file inside jar (`URISyntaxException` might be thrown when accessing it), copy it into a temporary one for accessing it from the dependent module."?

Make it clear,  OK  for me